README.md

```diff
@@ -1,2 +1,51 @@
 # in-stock-bot
-A simple python telegram bot checking if a product is in stock
+* [![GitHub Stars](https://img.shields.io/github/stars/MayNiklas/in-stock-bot.svg)](stargazers)
+
+* [![GitHub Issues](https://img.shields.io/github/issues/MayNiklas/in-stock-bot.svg)](issues)
+
+* [![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
```

Comment:
We are not having a build check on Travis yet - I would like to do it with GitHub Actions -> it's free.
This embedded icon is linking someone else's project 😅